.github/workflows/go-trust.yml

```diff
@@ -93,6 +93,7 @@ jobs:
       - name: Run ${{ matrix.task }} tests
         run: bin/task test-integration-${{ matrix.task }}
         env:
+          GOFLAGS: ${{ runner.debug == '1' && '-v' || '' }}
```

Comment:
My browser failed to display it by `View raw logs` option, so I downloaded it. Previous log archive was 62MB and 
and and changed one is 252MB. Should this be okay?

Reply:
That's only when tests are restarted in debug mode:

![CleanShot 2023-06-20 at 12 14 24@2x](https://github.com/FerretDB/FerretDB/assets/11512/a6330f15-a519-403c-8895-ec0b9cf8b502)

Reply:
Ah I see, I didn't know such option existed :see_no_evil: So it's no issue then :hugs: